src/styles/theme.js

```diff
@@ -0,0 +1,46 @@
+const calcRem = (size) => `${size / 16}rem`;
+
+const colors = {
+  black: "#1a1d23",
+  blue: "#0f4cd9",
+  white: "#ffffff",
+  darkgrey: "313439",
+  grey: "#484a4f",
+  lightgrey: "#ddddde",
+};
+
+const fontSizes = {
+  small: calcRem(8),
+  base: calcRem(16),
+  middle: calcRem(24),
+  big: calcRem(32),
+  bigger: calcRem(40),
+  biggest: calcRem(48),
+};
+
+const margins = {
+  small: calcRem(8),
+  base: calcRem(16),
+  middle: calcRem(24),
+  big: calcRem(32),
+  bigger: calcRem(40),
+  biggest: calcRem(48),
+};
+
+const paddings = {
+  small: calcRem(8),
+  base: calcRem(16),
+  middle: calcRem(24),
+  big: calcRem(32),
+  bigger: calcRem(40),
+  biggest: calcRem(48),
+};
```

Comment:
```suggestion
const padding = {
  small: calcRem(8),
  base: calcRem(16),
  middle: calcRem(24),
  big: calcRem(32),
  bigger: calcRem(40),
  biggest: calcRem(48),
};
```
복수형 변수명은 배열일 경우에 써야 할 것 같아요! 다른 변수명도 수정 부탁드려요!